doc/content/guides/getting-started/configuration.md

```diff
@@ -44,11 +44,13 @@ Before we go into the details of {{% tts %}}'s configuration, we'll take a look
 
 The `ports` section exposes {{% tts %}}'s ports to the world. Port `80` and `443` are mapped to the internal HTTP and HTTPS ports. The other ports have a direct mapping. If you don't need support for gateways and applications that don't support TLS, you can remove ports starting with `188`.
 
+In the `environment` section, we configure the databases used by the stack. We will set these to the CockroachDB and Redis instances that are defined in the `docker-compose.yml` above.
```

Comment:
`CockroachDB` or `Postgres`

Reply:
I can add this as a note, but we do not actually mention PostgreSQL at the getting started guide at all.

Reply:
Here would probably be a more appropriate place for this

https://github.com/TheThingsNetwork/lorawan-stack/blob/76baee22cb65b6e6f2cd3da781a0c09c797d6d9d/doc/content/guides/getting-started/configuration.md#L10-L12